.travis.yml

```diff
@@ -68,6 +68,7 @@ before_script:
 script:
     - if [ "$DISTRIB" == "ubuntu" ]; then python setup.py nosetests; fi
     - if [ "$DISTRIB" == "conda" ]; then nosetests; fi
+    - nosetests bin # to test jupyter notebook generator CLI
```

Comment:
I put this line because I could not get nosetest to scan the `bin` folder. Pytest did not have issues on that. But indeed I'll move everything to the notebook parsing file.